Masonry/MASViewAttribute.h

```diff
@@ -14,6 +14,8 @@
  */
 @interface MASViewAttribute : NSObject
 
+NS_ASSUME_NONNULL_BEGIN
+
 /**
  *  The view which the reciever relates to. Can be nil if item is not a view.
  */
```

Comment:
i mean, it _could_ be forced that the parameter is non-nil in the initializer, and the property still can become nullable - but as the `MASConstraintDelegateMock` explicitly passes `nil`, I went with making it nullable.